src/current/v25.1/create-logically-replicated.md

```diff
@@ -0,0 +1,115 @@
+- The [`REPLICATION` system privilege]({% link {{ page.version.version }}/security-reference/authorization.md %}#privileges).
+
+Use the [`GRANT SYSTEM`]({% link {{ page.version.version }}/grant.md %}) statement:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+GRANT SYSTEM REPLICATION TO user;
+~~~
+
+## Synopsis
+
+<div>
+{% include {{ page.version.version }}/ldr/create_logically_replicated_stmt.html %}
+</div>
+
+### Parameters
+
+Parameter | Description
+----------+------------
+`db_object_name` | The fully qualified name of the table on the source or destination cluster. Refer to [Examples](#examples).
+`logical_replication_resources_list` | A list of the fully qualified table names on the source or destination cluster to include in the LDR stream. Refer to the [LDR with multiple tables](#multiple-tables) example.
+`source_connection_string` | The connection string to the source cluster. Use an [external connection]({% link {{ page.version.version }}/create-external-connection.md %}) to store the source cluster's connection URI. To start LDR, you run `CREATE LOGICALLY REPLICATED` from the destination cluster.
+`logical_replication_create_table_options` | Options to modify the behavior of the LDR stream. For a list, refer to [Options](#options). **Note:** `bidirectional on` or `unidirectional`is a required option.
+
+## Options
+
+Option | Description
+-------+------------
+`bidirectional on` / `unidirectional` | (**Required**) Specifies whether the LDR stream will be unidirectional or bidirectional. With `bidirectional on` specified, LDR will set up two LDR streams between the clusters.
+`label` | Tracks LDR metrics at the job level. Add a user-specified string with `label`. For more details, refer to [Metrics labels]({% link {{ page.version.version }}/logical-data-replication-monitoring.md %}#metrics-labels).
+`mode` | Determines how LDR replicates the data to the destination cluster. Possible values: `immediate`, `validated`. For more details, refer to [LDR modes](#ldr-modes).
+
+## LDR modes
+
+_Modes_ determine how LDR replicates the data to the destination cluster. There are two modes:
+
+- `immediate` (default): {% include {{ page.version.version }}/ldr/immediate-description.md %}
+- `validated`: {% include {{ page.version.version }}/ldr/validated-description.md %}
+
+## Examples
+
+`CREATE LOGICALLY REPLICATED` will automatically create the specified source tables on the destination cluster. For unidirectional and bidirectional, you run the statement to start LDR on the destination cluster that does not contain the tables.
+
+### Unidirectional
+
+From the destination cluster of the LDR stream, run:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE LOGICALLY REPLICATED TABLE {database.public.destination_table_name} FROM TABLE {database.public.source_table_name} ON 'external://source' WITH unidirectional, mode=validated;
+~~~
+
+Include the following: 
+
+- Fully qualified destination table name.
```

Comment:
these mentions of fully qualified names could link to our name resolution docs: https://www.cockroachlabs.com/docs/v25.1/sql-name-resolution.html

Reply:
I'm definitely not going to forget this link in future now 😸